src/main/java/Turtle.java

```diff
@@ -0,0 +1,35 @@
+public class Turtle implements Drivable, Domesticatable, Tradable {
+
+    private int maxSpeed = 100;
+
+    @Override
+    public String sound() {
+        return "Meow! :)";
+    }
+
+    @Override
+    public void upgradeSpeed() {
+        this.maxSpeed += 3;
+    }
+
+    @Override
+    public void downgradeSpeed() {
+        this.maxSpeed--;
+    }
+
+    @Override
+    public int getMaxSpeed() {
+        return this.maxSpeed;
+    }
+
+    @Override
+    public int getPrice() {
+        return 31415;
+    }
+
+    @Override
+    public String toString(){
+        String id = super.toString();
```

Comment:
Unsure of what `super.toString();` should accomplish since there is no `super` class for Turtle (it just implements interfaces). If it pulls a memory ID for the object, I believe this might be confusing to see in a string format, and so a better implementation of `toString()` might be to attribute a name (unique) to each `Turtle` rather than referring to them by the arbitrary Java ID.